code/modules/wiremod/core/admin_panel.dm

```diff
@@ -0,0 +1,75 @@
+/// An admin verb to view all circuits, plus useful information
+/datum/admins/proc/view_all_circuits()
+	set category = "Admin.Game"
```

Comment:
we don't have this category. Use Adminbus

Reply:
done